CanOpen-and-Canfestivino-installation-guide.md

```diff
@@ -0,0 +1,302 @@
+```
+####Write your callback functions
+Using the exising callback functions `readSensorCallback` and `writeActuatorCallback` as a template, write your own callback functions. Give them descriptive names, making it clear which components they will be accessing. You're free to name the functions whatever you wish.
+> Received data is saved under the same variable as the name of your `Manufacturer Specific` object
+
+####Mapping callback functions
+In the `void setup()`, you must now map your callback functions to the corrosponding object indices and subindices, much like as is done in the example. The syntax is:
+
+```
+RegisterSetODentryCallBack([index], [subindex], [callback function])
+```
+Where `index` is the index number of the Manufacturer Specific object you defined in the .od/.eds.
+`subindex` should be 0, unless you defined a subindex
+`callback function` is the callback function you defined earlier in the step above.
+
+In the example, these are enclosed in print statements, as these will return an error code if something went wrong (otherwise simply `0` if it was successful.
+
+###Final Steps
+Now simply compile the sketch and upload it to the Arduino and it should start sending it's heartbeat over the CANBUS network.
+
+##Preparing the master node
+
+> As explained in the "Editing a slave node .EDS" section, the master node cannot request or change the slave node's .EDS file at runtime. Therefore we need to make sure it has all necessary information beforehand. The major downside to this is that any changes made to a slave node or to the network must also be reflected in all of the corresponding files.
+> 
+>  This also means that you must also manually keep track of all nodes ID's in your network, as there are no built-in checks for conflicting ID's.
+> 
+> It is especially important in the following steps that Node ID's correspond to the ID defined in the .OD
+> 
+> Our recommendation is to have some kind of system that makes it easy to find the ID of a node, such putting it filenames.
+
+###Generating the master OD/EDS with Networkedit
+Make sure you're in the `objdictgen` folder. If not, use the following command:
+
+```
+cd objdictgen
+```
+
+Then run networkedit with the following command:
+
+```
+python2 networkedit.py
+```
+
+Go to `File > Open` and open the folder containing your slave node `.eds` files. This should open `0x00 MasterNode` on the left colomn.
+Optionaly, click on `0x1000-0x1029 Communication Parameters` and then `0x1018 Identity` and fill in the Vendor ID, Product code etc.
+
+We will now add the slave nodes. Go to `Network > Add slave node` and fill in the exact same `Slave Name` and `Slave Node ID` as you did in the previous section. Click on the dropdown arrow on `EDS file` and choose the appropriate file from the list. If the `.eds` is not in the folder you chose to open, you can import this with the `Import EDS` button, then use the dropdown menu to choose the correct file.
```

Comment:
We will now add the slave nodes -> We going to add the slave nodes.

Reply:
Is correct engels :P